common/azure-test-watcher/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-test-watcher</artifactId>
+  <version>1.0.0-beta.1</version>
```

Comment:
Should this line have that x-version-update metadata?

Reply:
This won't be managed by the versioning script, it should be treated the same as the code-quality-reports and jacoco projects.